odoo/addons/test_lint/tests/test_manifests.py

```diff
@@ -2,13 +2,19 @@
 
 import logging
 from ast import literal_eval
-from os.path import join as opj
+from os.path import join as opj, splitext
+from pathlib import Path
 
 from odoo.modules import get_modules
 from odoo.modules.module import _DEFAULT_MANIFEST, module_manifest, get_module_path
 from odoo.tests import BaseCase
 from odoo.tools.misc import file_open, file_path
 
+DFTL_MANIFEST_DATA_KEYS = ["data", "demo"] + [k for k in _DEFAULT_MANIFEST if k.endswith("_xml")]
+MANIFEST_DATA_DIRS = ["data", "datas", "demo", "demos", "report", "reports", "security", "template", "templates", "view", "views", "wizard", "wizards"]
+MANIFEST_DATA_EXTS = [".csv", ".xml"]
+MANIFEST_DATA_NOT_USED_EXCLUDE_KEY = "not_used_exclude"
```

Comment:
I wouldn't add a new key, but instead move these files elsewhere. `init_data` or similar.

@Julien00859 ?

Reply:
I agree with @kmagusiak, I don't like this new key. I would rather we wait for #211150 and then move the remaining files outside of any `MANIFEST_DATA_DIRS` folder. I don't think we need a new folder, and I don't want to spent too long thinking of a good name. They could very well be moved in the module's root directory (next to `__manifest__.py`, or closer to the file that use them (i.e. inside `controllers/` or `models/`), or even inside `static/` if the module is in community.